evaporate.js

```diff
@@ -21,6 +21,8 @@
 (function () {
   "use strict";
 
+  var Promise = require("es6-promise-polyfill").Promise;
```

Comment:
If developers want to implement a Promise polyfill, they are free to do that, particularly if they're targeting Edge, but the Polyfill is not a required dependency of EvaporateJS.